Core/HistoManager.cpp

```diff
@@ -329,7 +331,7 @@ void HistoManager::writeError(const char* nameOfHistogram, const char* messageEn
 {
     std::string histName (nameOfHistogram);
     bool histExists =  (fErrorCounts.find(histName) != fErrorCounts.end());
-    if (!histExists) 
+    if (!histExists)
     {
         fErrorCounts.insert(histName);
         std::cout << "!!![Error]!!!  -  Histogram with name " << histName << " " << messageEnd << std::endl;
```

Comment:
By the way, could you change `cout` to `G4cout` here?

Reply:
ok